ideas.html

```diff
@@ -0,0 +1,156 @@
+                <li>Build a monitoring and rules engine (or adaptor for an existing one) that can be used for operating an ILP node</li>
+                <li>Build an data collection and aggreagtion service for ILP data that can be used to monitor the health of the network.</li>
+            </ul>
+
+            <h4>Tech</h4>
+            <p>Javascript for updates to ILP-kit. Any for other deliverables.</p>
+
+            <h4>Mentors</h4>
+            <p>TBD</p>
+
+
+            <hr/>
+
+            <h3>ILP-enable a Ledger</h3>
+
+            <p>Interledger has specific service requirements from any payment network/ledger that becomes part of the network. The most common way to ILP-enable a ledger is to write a plugin/adaptor for that ledger that conforms to the Interledger specifications</p>
+
+            <p>The community has been particulalry excited about integrating the Bitcoin Lightning and Ethereum Raden netwokrs into the Interledger so experience with eithr of these would be a big bonus.</p>
+
+            <h4>Requirements</h4>
+
+            <p>Build a new Javascript ledger plugin that confirms to the <a href="https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md">ledger plugin specification</a> or build a Java ledger adaptor based on the <a href="https://github.com/interledger/java-ilp-core/blob/development/src/main/java/org/interledger/ilp/ledger/LedgerAdaptor.java">Java interface</a>.</p>
+
+            <h4>Tech</h4>
+            <p>Javascript or Java</p>
+
+            <h4>Mentors</h4>
+            <p>For Java adaptors: <a href="https://github.com/adrianhopebailie">Adrian Hope-Bailie</a></p>
+            <p>For Javascript plugins: <a href="https://github.com/emschwartz">Evan Schwartz</a></p>
+
+            <hr/>
+
+            <h3>Java Implementations</h3>
+            <p>The Javascript implementations of Interledger are more mature than the Java equivalents.</p>
+
+            <h4>Requirements</h4>
+            <p>Pick any Java component that needs work to get to feature parity with the JavaScript and go...</p>
+
+            <h4>Tech</h4>
+            <p>Java</p>
+
+            <h4>Mentors</h4>
+            <p><a href="https://github.com/adrianhopebailie">Adrian Hope-Bailie</a></p>
+
+            <hr/>
+
+            <h3>Any other implementations</h3>
+            <p>Various ILP-related componenets have been developed by community memebers in Python, Haskell, Go and other langauges. If you'd like to implement ILP in a new langauge we'll find a mentor to help you do it!</p>
+
+            <h4>Requirements</h4>
+            <p>Implement an ILP Client in a language other than Javascript or Java.</p>
+
+            <h4>Tech</h4>
+            <p>Any</p>
+
+            <h4>Mentors</h4>
+            <p>TBD</p>
+
+            <hr/>
+
+            <h3>Build Micropayments Into an App or Protocol</h3>
```

Comment:
Don't think there's anything to be read into the ordering tbh but. Otherwise any one that proposes new projects is just going to ask for their to be put at the top everytime.

From a project priorities perspective I think the ILP-kit related stuff is most important which is why I started with those.